package.json

```diff
@@ -18,8 +18,8 @@
     "cypress:open": "cypress open",
     "cypress:run": "cypress run",
     "test": "is-ci \"test:coverage\" \"test:watch\"",
-    "test:coverage": "jest --coverage",
-    "test:watch": "jest --watch",
+    "test:coverage": "jest --coverage --env=jsdom",
+    "test:watch": "jest --watch --env=jsdom",
```

Comment:
If you need the `jsdom` environment in the test, please update the regex in the `test/jest.client.js` file to match the file and update the regex in the `test/jest.server.js` file to exclude the file.

Reply:
Done.